packages/main/src/ResponsivePopover.js

```diff
@@ -131,6 +139,18 @@ class ResponsivePopover extends Popover {
 		}
 	}
 
+	toggle(opener) {
```

Comment:
I can't put a comment on an old line, but in Popover.js in the `reposition` function there is a typo: streching => stretching